src/ui/buttons/buttonDesigns/fadeOutLogoShowLabelText.jsx

```diff
@@ -0,0 +1,121 @@
+        ANIMATION_LABEL_CONTAINER:  'personalized-label-container',
+        PAYPAL_BUTTON_LABEL:        CLASS.BUTTON_LABEL
+    }
+};
+
+export function FadeoutLogoShowLabelTextComponent({ designLabelText } : LabelOptions) : ChildType {
+    const CONTAINER_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_CONTAINER;
+    const LABEL_CLASS = FADEOUT_LOGO_SHOW_LABEL_TEXT_CONFIG.cssClasses.ANIMATION_LABEL_CONTAINER;
+   
+    return (
+        <Fragment>
+            <div class={ LABEL_CLASS } data-design-experiment='104530'> <span>{designLabelText}</span></div>
+            <style innerHTML={ `
+                .${ CLASS.DOM_READY } .${ CONTAINER_CLASS } img.${ LOGO_CLASS.LOGO }{
+                    position: relative;
+                }
+                
+                .${ CONTAINER_CLASS } .${ LABEL_CLASS } {
+                    position: fixed;
+                    opacity: 0; 
+                    color: #142C8E;
+                    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+                    font-size: 10px;
+                }
+            ` } />;
+        </Fragment>
+    );
+}
+
+// Returns label container if the button sizes match
+export const getFadeoutLogoShowLabelTextProps = function(document, configuration) : DivideLogoAnimationProps | null {
+    const { ANIMATION_CONTAINER, ANIMATION_LABEL_CONTAINER, PAYPAL_BUTTON_LABEL, PAYPAL_LOGO } = configuration.cssClasses;
+    const { tiny, medium } = configuration;
+    // get the animation main container to force specificity( in css ) and make sure we are running the right animation
+    const animationContainer = (document && document.querySelector(`.${ ANIMATION_CONTAINER }`)) || null;
+    if (!animationContainer) {
+        return null;
+    }
+
+    // return null if animation should not be played for the button size
+    const animationContainerWidth = animationContainer.offsetWidth;
+    if (animationContainerWidth < tiny.min || animationContainerWidth > medium.max) {
+        // remove label element from dom
+        animationContainer.querySelector(`.${ ANIMATION_LABEL_CONTAINER }`).remove();
+        return null;
+    }
+
+    // get the label container that animation will be applied to
+    const paypalLabelContainerElement = animationContainer.querySelector(`.${ PAYPAL_BUTTON_LABEL }`) || null;
+    // get starting position for element so it doesn't jump when animation begins
+    const paypalLogoElement = (paypalLabelContainerElement && paypalLabelContainerElement.querySelector(`.${ PAYPAL_LOGO }`)) || null;
+    const paypalLogoStartingLeftPosition = paypalLogoElement
+        ? `${ (paypalLogoElement.offsetLeft / paypalLabelContainerElement.offsetWidth) * 100 }`
+        : '44.5';
+
+    return {
+        paypalLabelContainerElement,
+        paypalLogoStartingLeftPosition
+    };
+};
+
+export function getFadeoutLogoShowLabelTextAnimation (animationProps, animationConfig) : void | null {
```

Comment:
can we add type to the arguments here?

Reply:
Hey @jrod1032 this change is outdated, please check again